app/views/users/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>This displays lists of users</h1>
+<% @users.each do |user|  %>
+
+  <div class="card mb-2" >
+    <div class="d-flex justify-content-between mb-3">
+      <div class="m-5">
+        <%= image_tag user.photo, :alt => "user_photo", :height => "150px", :width => "150px" %>
+      </div>
+      <div class="m-5 flex-grow-1">
+        <article class="card-body " style="width: 18rem;"> 
+          <h1><a><%= link_to user.name, user %></a>
+          </h1>
+          </article>
+      </div>
+      <div class="mb-2 mr-5 d-flex flex-wrap align-content-end">
+          <h5>Number of Posts: <%= user.post_counter %>
+          </h5>
+        </div>
+    </div>
+  </div>
+
+<% end %>
```

Comment:
- [OPTIONAL] You can use a partial ` _user.html.erb` here as it is a better practice instead of using @objects.each
eg: 
```ruby
 # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```

Reply:
Thank you for your feedback and suggestion.